MacawTests/MacawSVGTests.swift

```diff
@@ -707,4 +784,62 @@ class MacawSVGTests: XCTestCase {
     func testPserversGrad03() {
         validateJSON("pservers-grad-03-b-manual")
     }
+    
+    func getImage(from svgName: String) -> MImage {
+        let bundle = Bundle(for: type(of: TestUtils()))
+        do {
+            let node = try SVGParser.parse(resource: svgName, fromBundle: bundle)
+            
+            var frame = node.bounds
+            if frame == nil, let group = node as? Group {
+                frame = Group(contents: group.contents).bounds
+            }
+            
+            let image = node.toNativeImage(size: frame?.size() ?? Size.init(w: 100, h: 100))
+            return image
+        } catch {
+            XCTFail(error.localizedDescription)
+        }
+        
+        XCTFail()
+        return MImage()
+    }
+    
+    func saveImage(image: MImage, fileName: String) {
+        #if os(OSX)
+        guard let data = image.tiffRepresentation else {
+            return
+        }
+        #endif
+        
+        #if os(iOS)
+        guard let data = image.pngData() else {
+            return
+        }
+        #endif
+        
+        let _ = writeToFile(data: data, fileName: "\(fileName).png")
+    }
+    
+    fileprivate func setupTestFolderDirectory() {
+        guard let myDocuments = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else {
+            return
+        }
+        
+        let testDirectoryPath = myDocuments.appendingPathComponent("\(testFolderName)")
+        
+        do {
+            if !multipleTestsWillRun {
+                try FileManager.default.removeItem(at: testDirectoryPath)
+            }
+            
+            var isDirectory: ObjCBool = ObjCBool(true)
+            if !FileManager.default.fileExists(atPath: testDirectoryPath.absoluteString, isDirectory: &isDirectory) {
+                try FileManager.default.createDirectory(at: testDirectoryPath, withIntermediateDirectories: true, attributes: .none)
+            }
+        } catch {
+            return
```

Comment:
It's better to log error or let it break. Otherwise it can start working wrong and we wouldn't know it.

Reply:
done